tests/signature_verification.rs

```diff
@@ -6,74 +6,72 @@
 
 //! Test for signature verification.
 
-use std::process::Command;
-
+use common::KBC;
 use image_rs::image::ImageClient;
+use rstest::rstest;
+use serial_test::serial;
 
 mod common;
 
 /// Name of different signing schemes.
 const SIMPLE_SIGNING: &str = "Simple Signing";
 const NONE_SIGNING: &str = "None";
 
-/// Script for preparing Simple Signing GPG signature file.
-const SIGNATURE_SCRIPT: &str = "scripts/install_test_signatures.sh";
-
-/// Parameter `decrypt_config` provided for `ImageClient`.
-const AA_PARAMETERS: &str = "provider:attestation-agent:sample_kbc::null";
-
 struct TestItem<'a, 'b, 'c> {
     image_ref: &'a str,
     allow: bool,
     signing_scheme: &'b str,
     description: &'c str,
 }
 
+/// Four test cases.
+const TESTS: [TestItem; 4] = [
```

Comment:
Do these tests need to run as `root`? If so, we should consider using the macros defined below to skip if not running as that user:

- https://github.com/confidential-containers/image-rs/blob/main/src/snapshots/occlum/unionfs.rs#L235